docs/operate/control/headless-app.md

```diff
@@ -5,5 +5,532 @@ weight: 30
+
+   {{< tabs >}}
+   {{% tab name="Python" %}}
+
+   The `connect()` function will resemble the following:
+
+   ```python {class="line-numbers linkable-line-numbers" data-line="4,7,9"}
+   async def connect():
+       opts = RobotClient.Options.with_api_key(
+           # Replace "<API-KEY>" (including brackets) with your machine's API key
+           api_key='<API-KEY>',
+           # Replace "<API-KEY-ID>" (including brackets) with your machine's API key
+           # ID
+           api_key_id='<API-KEY-ID>'
+       )
+       return await RobotClient.at_address('ADDRESS FROM THE VIAM APP', opts)
+   ```
+
+   Copy not just this function but the entire code snippet on the **CONNECT** tab if you'd like to test your connection with some basic API calls.
+
+   {{% /tab %}}
+   {{% tab name="Go" %}}
+
+   The machine connection function will resemble the following:
+
+   ```go {class="line-numbers linkable-line-numbers" data-line="3,7,11"}
+   machine, err := client.New(
+       context.Background(),
+       "ADDRESS FROM THE VIAM APP",
+       logger,
+       client.WithDialOptions(rpc.WithEntityCredentials(
+       // Replace "<API-KEY-ID>" (including brackets) with your machine's API key ID
+       "<API-KEY-ID>",
+       rpc.Credentials{
+           Type:    rpc.CredentialsTypeAPIKey,
+           // Replace "<API-KEY>" (including brackets) with your machine's API key
+           Payload: "<API-KEY>",
+       })),
+   )
+   ```
+
+   Copy not just this function but the entire code snippet on the **CONNECT** tab if you'd like to test your connection with some basic API calls.
+
+   {{% /tab %}}
+   {{% tab name="C++" %}}
+
+   The machine connection code will resemble the following:
+
+   ```cpp {class="line-numbers linkable-line-numbers" data-line="1,5,7"}
+   std::string host("ADDRESS FROM THE VIAM APP");
+   DialOptions dial_opts;
+   dial_opts.set_type("api-key");
+   // Replace "<API-KEY-ID>" with your machine's API key ID
+   dial_opts.set_entity("<API-KEY-ID>");
+   // Replace "<API-KEY>" with your machine's API key
+   Credentials credentials("<API-KEY>");
+   dial_opts.set_credentials(credentials);
+   boost::optional<DialOptions> opts(dial_opts);
+   Options options(0, opts);
+
+   auto robot = RobotClient::at_address(host, options);
+   ```
+
+   Copy not just this function but the entire code snippet on the **CONNECT** tab if you'd like to test your connection with some basic API calls.
+
+   {{% /tab %}}
+   {{< /tabs >}}
+
+{{< alert title="Caution" color="caution" >}}
+Do not share your machine part API key or machine address publicly.
+Sharing this information could compromise your system security by allowing unauthorized access to your machine, or to the computer running your machine.
+{{< /alert >}}
+
+## Write your control script
+
+Save the code on the computer or SBC where you'll run it.
+Edit it to add your control logic.
```

Comment:
I think these can be cut? The main important thing is to point people at the APIs here I think? And then showing the examples seems good. I was vaguely wondering if https://docs.viam.com/#program-any-device would come in useful anywhere but with the languages more separated out it seems awkward. 
```suggestion
```

Reply:
Hmm I could use a version of program any device just with TS and Flutter stripped out... I'm thinking that full examples are better because they show getting the resource from the robot etc., and don't leave the user wondering how it all ties together, but the program any device content does showcase the APIs nicely. Will have a think.

Reply:
As far as deleting these lines, I think we've gotten feedback at some point(s) that people were confused about not just how the code talks to the machine, but also where the code goes and where the SDK is installed etc. Maybe that was only newer hobbyists though and this is obvious.